src/libraries/System.IO.Hashing/src/System.IO.Hashing.csproj

```
@@ -4,8 +4,10 @@
     <Nullable>enable</Nullable>
     <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
     <IsPackable>true</IsPackable>
+    <!-- Disable package on release/6.0-maui branch, revert before merging back into release/6.0
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <ServicingVersion>1</ServicingVersion>
+    -->
```

revert the branch specific changes:
```suggestion
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
    <ServicingVersion>1</ServicingVersion>
```